docs/src/pages/components/data-grid/getting-started/migration-v4.md

```diff
@@ -1,26 +1,29 @@
 ---
-title: Data Grid - Migration from v4 to v5
+title: MUI X - Migration from v4 to v5
 ---
 
-# Data Grid - Migration from v4 to v5
+# MUI X - Migration from v4 to v5
 
 <p class="description">MUI X v5 is now stable!</p>
 
 ## Introduction
 
-This a reference for upgrading your site from MUI X v4 to v5. MUI X v5 is fully compatible with MUI Core v5 and can be used with MUI Core v4 with some additional steps.
+This a reference for upgrading your site from MUI X v4 to v5.
+MUI X v5 is fully compatible with MUI Core (includes Material UI) v5 and can be used with MUI Core v4 with some additional steps.
 Most breaking changes are renaming of CSS classes or variables to improve the consistency of the grid.
 
-## Using MUI X v5 with MUI Core v4
+## Migrating MUI Core from v4
 
-We strongly recommend you to migrate both MUI X and MUI Core to v5.
-Depending on the complexity of the application however, this might not be possible.
-An alternative is to install MUI Core v4 alongside to v5, and to configure them to avoid conflicts.
-This can be achieved with the following steps:
+> We strongly recommend you [migrate MUI Core to v5](/guides/migration-v4/) when using MUI X v5.
+> However, this might not be possible, depending on the complexity of the application.
+> The alternative is to install MUI Core v5 and configure it to keep MUI Core v4 running alongside.
 
-1. First, make sure you have MUI X v5 installed. If not, install it with these [instructions](/components/data-grid/getting-started/#installation).
+### Using MUI Core v4 with v5
```

Comment:
I think that the previous header was clearer. Also, "site", it's feels a bit off. Mostly apps that are built with React.

Reply:
You're right, I used "site" to be consistent with the introduction, but I wasn't certain about it either. 
The main point of the suggestions though, is that I felt "v4 with v5" was inverted in their order, since the guide is about how to use v5.

Reply:
Yeah, maybe the opposite would work better 👍 

```suggestion
### Using MUI Core v5 with v4
```